ReactTask/src/tests/SignUpForm.test.tsx

```diff
@@ -8,11 +7,11 @@ import { FormSignUp } from '../components';
 import theme from '../theme';
 
 jest.mock('./../redux/patient/patientOperations', () => ({}));
-
+type Default = string;
```

Comment:
What for do you need type, just use sting

Reply:
Ok, thanks information